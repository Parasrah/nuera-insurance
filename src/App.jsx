import React from 'react'
import { StyleSheet, css } from 'aphrodite'

import { Table } from '@/Table'
import { useTable } from '@hook/table'

function App () {
  const tableHooks = useTable()

  return (
    <div className={css(styles.container)}>
      <Table {...tableHooks} />
    </div>
  )
}

const styles = StyleSheet.create({
  container: {
    margin: 0,
    height: '100%',
    width: '100%',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center'
  }
})

export { App }
