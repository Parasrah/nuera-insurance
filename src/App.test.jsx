/* globals expect, jest, describe, test */
import React from 'react'
import renderer from 'react-test-renderer'

import { App } from '@/App'
jest.mock('@/Table')

describe('#render', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<App />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
