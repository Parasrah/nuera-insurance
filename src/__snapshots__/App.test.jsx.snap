// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="table"
  />
</div>
`;
